src/features/twitter/controllers/twitter-controller.test.ts

```diff
@@ -69,15 +78,30 @@ describe('[Controller] Twitter', () => {
   });
 
   describe('login', () => {
-    it('should be return 401', () => {
-      const { authController, next, req, res } = makeSut();
+    it('should be return link', () => {
+      const { authController, loginTwitterService, next, req, res } = makeSut();
 
-      req.headers.authorization = undefined;
+      req.headers.authorization = 'Bearer token';
 
+      const serviceSpy = vi
+        .spyOn(loginTwitterService, 'execute')
+        .mockReturnValue('url');
       authController.login(req, res, next);
 
-      expect(res.status).toHaveBeenCalledWith(401);
-      expect(res.json).toHaveBeenCalledWith({ message: 'Unauthorized' });
-    });
+      expect(serviceSpy).toHaveBeenCalledWith({
+        authorization: 'Bearer token',
+      });
+      expect(res.json).toHaveBeenCalledWith('url');
+    }),
+      it('should be return 401', () => {
```

Comment:
Toma cuidado com o "be" ele não é obrigatório e não faz muito sentido gramaticalmente da forma que esta usando  bb.
```suggestion
      it('should return 401 if authorization header is missing', () => {
```

Reply:
ta bom, muito obrigado nico <3